patch/0157-platform-x86-mlx-platform-Make-activation-of-some-dr.patch

```diff
@@ -0,0 +1,120 @@
+From b384a287a5732f7ea3b6e0b13b1aa6ba0d70c440 Mon Sep 17 00:00:00 2001
+From: Vadim Pasternak <vadimp@nvidia.com>
+Date: Mon, 14 Feb 2022 09:46:16 +0200
+Subject: [PATCH platform-next 1/8] platform/x86: mlx-platform: Make activation
+ of some drivers conditional
+
```

Comment:
Please add the upstream commit hash somewhere.

Reply:
Hi
we have it in the description `0157-platform-x86-mlx-platform-Make-activation-of-some-dr.patch https://github.com/torvalds/linux/commit/e05d6b658fcd98bd3a09220cb5701e40269a0e57`